app/src/main/kotlin/com/hippo/ehviewer/ui/scene/SearchBarScene.kt

```diff
@@ -231,102 +240,111 @@ fun SearchBarScreen(
         }
     }
 
-    var searchbarOfs by searchBarOffsetY
-    Scaffold(
-        topBar = {
-            SearchBar(
-                modifier = Modifier.fillMaxWidth().layout { measurable, constraints ->
-                    val placeable = measurable.measure(constraints)
-                    layout(placeable.width, placeable.height) {
-                        placeable.placeRelative(0, searchbarOfs)
+    val searchbarOfs by rememberUpdatedState(searchBarOffsetY)
+    val scrollAwayModifier = if (!active) {
+        Modifier.layout { measurable, constraints ->
+            val placeable = measurable.measure(constraints)
+            layout(placeable.width, placeable.height) {
+                placeable.placeRelative(0, searchbarOfs)
+            }
+        }
+    } else {
+        Modifier
+    }
+    Box(modifier = Modifier.fillMaxSize()) {
+        Scaffold(
+            topBar = {
+                // Placeholder, fill immutable SearchBar padding
+                Spacer(modifier = Modifier.height(112.dp))
```

Comment:
Better to not used fixed value here as status bar & display cutout insets may differ between devices.

Reply:
Then let it padding a safeDraw?

Reply:
> Then let it padding a safeDraw?

SGTM.